### DISCUSSION TITLE
Custom training

### DISCUSSION THREAD

**Discussion by composition-cassidy** [Anjok07/ultimatevocalremovergui#425] [Q&A]
Can I do my own custom training to create my own models? If so, how do I do that?

**Comment by KimberleyJensen** [comment]
@composition-cassidy you should visit the pages of the models where you can find the training code

demucs - https://github.com/facebookresearch/demucs
mdx-net - https://github.com/kuielab/mdx-net / https://github.com/KimberleyJensen/mdx-net
vocal remover - https://github.com/tsurumeso/vocal-remover
